src/test/fuzz.cpp

```
@@ -227,4 +227,16 @@ genfuzz(std::string const& filename)
         }
     }
 }
+// default "TRANSACTION" if unrecognized or unspecified
```

no silent failures please: if not recognized -> help